examples/gno.land/r/demo/inner/inner_test.gno

```diff
@@ -0,0 +1,30 @@
+package inner
+
+import (
+	"std"
+	"testing"
+
+	"gno.land/p/demo/testutils"
+)
+
+var (
+	oc std.Address = testutils.TestAddress("origin caller")
+	or string      = "gno.land/r/demo/outer_realm"
+)
+
+func TestOuterCallInner(t *testing.T) {
+	std.TestSetOrigCaller(oc)
+	shouldEQ(t, std.GetOrigCaller(), oc)
+
+	std.TestSetPrevRealm(or)
+	pr := std.PrevRealm()
+	shouldEQ(t, pr.Addr(), std.DerivePkgAddr(or))
+	shouldEQ(t, pr.PkgPath(), or)
+	shouldEQ(t, std.GetOrigCaller(), oc)
+}
+
+func shouldEQ(t *testing.T, got, want interface{}) {
```

Comment:
What do you think about naming this as `assertEqual`, or `assertE` to follow convention from other `.gno` assertion checks?
This should probably be in a package of its own, but it is out of the scope of this PR